.github/CHANGELOG.md

```diff
@@ -1,38 +1,60 @@
-## Release 0.7.0a4 (development release)
+## Release 0.8.0a4 (development release)
+
+### New features since the last release
+
+
+### Bug fixes
+
+
+### Improvements
+
+
+### Documentation changes
+
+
+### Contributors
+
+This release contains contributions from (in alphabetical order):
+
+See full commit details ...
+
+---
+
+## Release 0.8.0a4 (current release)
 
 ### New features since the last release
 
 * Node and edge coloring can now be done based on any attribute and personalized colors can be defined via a dictionary: [#32](https://github.com/XanaduAI/flamingpy/pull/32)(backward compatible)
- * The `EGraph` plot legend is not limited to the "state" attribute of the node but to any attribute.
+* The `EGraph` plot legend is not limited to the "state" attribute of the node but to any attribute. [#32](https://github.com/XanaduAI/flamingpy/pull/32)(backward compatible)
```

Comment:
```suggestion
* The `EGraph` plot legend is not limited to the "state" attribute of the node but to any attribute. [#32](https://github.com/XanaduAI/flamingpy/pull/32) (backward incompatible)
```
Technically this is not backward compatible, because it requires color_nodes to be a new data type  -- tuple(str, dict)